### PR TITLE
Updating Get-CsPhoneNumberAssignment

### PR DESCRIPTION
**Pull request by JensTrier**
Updating Get-CsPhoneNumberAssignment with new output parameter introduced in TPM 5.7.0

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 1]
```diff
@@ -453,6 +455,9 @@ Boolean stating if updating of the location assigned to the phone number is allo
 ### NetworkSiteId
 This parameter is reserved for internal Microsoft use.
 
+### NumberSource
+The source of the phone number. Online for phone numbers assigned in Microsoft 365 and OnPremises for phone number assigned in AD on-premises and synchronized into Microsoft 365.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:459]:
```suggestion
The source of the phone number. Online for phone numbers assigned in Microsoft 365 and OnPremises for phone numbers assigned in AD on-premises and synchronized into Microsoft 365.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@JensTrier, small edit, please.

**Comment by JensTrier** [comment]
> @JensTrier, small edit, please.

@JohanFreelancer9 thanks & done!

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd

**Pull request merged**